--- ZendFramework-1.7.6/library/Zend/Gdata/YouTube.php~	2009-03-10 22:11:26.000000000 +0200
+++ ZendFramework-1.7.6/library/Zend/Gdata/YouTube.php	2009-03-10 22:13:01.898984500 +0200
@@ -687,7 +687,7 @@
     public function getFriendActivityForCurrentUser()
     {
         if (!$this->isAuthenticated()) {
-            require_once 'Zend/Gdata/YouTube/App/Exception.php';
+            require_once 'Zend/Gdata/App/Exception.php';
             throw new Zend_Gdata_App_Exception('You must be authenticated to ' .
                 'use the getFriendActivityForCurrentUser function in Zend_' .
                 'Gdata_YouTube.');
--- ZendFramework-1.7.6/library/Zend/Service/Audioscrobbler/Exception.php~	2009-03-10 22:15:39.811374312 +0200
+++ ZendFramework-1.7.6/library/Zend/Service/Audioscrobbler/Exception.php	2009-03-10 22:15:31.689629002 +0200
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Zend Framework
+ *
+ * LICENSE
+ *
+ * This source file is subject to the new BSD license that is bundled
+ * with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://framework.zend.com/license/new-bsd
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@zend.com so we can send you a copy immediately.
+ *
+ * @category   Zend
+ * @package    Zend_Service
+ * @subpackage Audioscrobbler
+ * @copyright  Copyright (c) 2005-2008 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ * @version    $Id$
+ */
+
+/**
+ * @category   Zend
+ * @package    Zend_Service
+ * @subpackage Audioscrobbler
+ * @copyright  Copyright (c) 2005-2008 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+class Zend_Service_Audioscrobbler_Exception extends Exception
+{
+}
